css/style.css

```diff
@@ -0,0 +1,461 @@
+body {
+  background-color: #f4f5f6;
+  color: #333;
+  font-family: Arial, Helvetica, sans-serif;
+  overflow-x: hidden;
+}
+
+.text-white {
+  color: white;
+}
+
+.bg-white {
+  background-color: white;
+}
+
+.top-header {
+  height: 95px;
+  background-color: red;
+  padding: 0 10px 15px;
+}
+
+.family-roboto {
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.container-fluid {
+  max-width: 1310px;
+  margin: 0 auto;
+  padding: 0;
+}
+
+.header-list li {
+  font-size: 18px;
+}
+
+.date {
+  margin-top: 35px;
+}
+
+.nav-sub {
+  margin-top: 30px;
+}
+
+.btn-in:hover {
+  color: white;
+  text-decoration: none;
+}
+
+.btn-sub {
+  padding: 7px 12px;
+  color: red;
+}
+.btn-sub:hover {
+  color: red;
+  text-decoration: none;
+}
+
+.header-nav {
+  padding: 15px 160px 15px 0;
+  border-bottom: 2px solid #d9d9d9;
+  height: 56px;
+  background-color: white;
+}
+.header-nav::before {
+  position: absolute;
```

Comment:
`SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._